runtime/runtime-params-estimator/emu-cost/ci.sh

```diff
@@ -12,7 +12,9 @@ docker run \
      --cap-add=SYS_PTRACE --security-opt seccomp=unconfined \
      -i -t rust-emu \
      /bin/bash -c "
+set -ex
 cd /host/${srcdir}/runtime/runtime-params-estimator
+pushd test-contract && ./build.sh && popd
```

Comment:
Personally I prefer a sub shell to not worry about the script's `pwd`. 
```suggestion
$(cd test-contract && ./build.sh)
```